timecog/timecog.py

```diff
@@ -21,7 +21,8 @@
 ]
 
 time_in_regeces = [
-    r'^\s*((?:-?\d+ ?(?:m|h|d|w|y|s)\w* ?)+)\b (.*)$'
+    r'^\s*((?:-?\d+ ?(?:m|h|d|w|y|s)\w* ?)+)\b (.*)$',
+    r'^\s*((?:-?\d+ ?(?:m|h|d|w|y|s)\w* ?)+|now)\b\s*\|\s*((?:-?\d+ ?(?:m|h|d|w|y|s)\w* ?)+)\b (.*)$',
```

Comment:
Please add comments that are a brief summary of these regexes, maybe an example or two of what they match.